src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -85,21 +88,28 @@ public EventConsumer createConsumer(DeltaTargetContext context) throws IOExcepti
       .setProjectId(project)
       .build()
       .getService();
-    // TODO: make bucket optional
-    Bucket bucket = storage.get(conf.stagingBucket);
+
+    String stagingBucketName = conf.stagingBucket == null ? null : conf.stagingBucket.trim();
```

Comment:
shall me make it to lowerCase too ? GCS bucket name only allow lower case

Reply:
yeah making lower case on line 96 below.